boards/qemu_i486_q35/Makefile

```
@@ -38,7 +39,7 @@ QEMU_BASE_CMDLINE := \
 run: $(TARGET_PATH)/release/$(PLATFORM).elf
 	@echo
 	@echo Reordering ELF file sections
-	@objcopy $<
+	@$(OBJCOPY_PATH)objcopy  $<
```

I think this is an unrelated change (I don't think this variable is anywhere in-tree; local env variable?)

---

This prevents Mac uses from constantly changing this in the Makefile :) If you don't set this, it works as it worked before this modification, if you add it to the environment, it prepends a path.

---

```suggestion
	@$(OBJCOPY)  $<
```

Can we just do that? That variable is already defined in the build system for this purpose [unless there's something weirder going on with qemu that requires a _different_ objcopy that we're otherwise using to normally build]